Assets/Scripts/Combat/Hitbox.cs

```diff
@@ -17,5 +17,10 @@ void OnTriggerEnter2D(Collider2D other)
         {
             other.GetComponent<Enemy>().TakeDamage(attackData.damage, transform.position, attackData.knockback);
         }
+        if( isEnemyHitbox && other.CompareTag("Player"))
+        {
+            // TODO - Add logic to deal damage to player
```

Comment:
A better pattern might be to create a common `Health` component that both the Enemy and Player share, that way `Hitbox` wouldn't care about what it is hitting. However this increases complexity as a more generalized solution can create weird edge cases, like the player hitting itself. This can be solved usually by adjusting the layer collision matrix in Physics2D settings, but again that does make things more complicated.